device_independent_test/entanglement.py

```diff
@@ -1,103 +1,69 @@
 import numpy as np
 from qiskit import IBMQ, QuantumCircuit, execute
-from qiskit.tools.monitor import job_monitor
-from qiskit.providers.ibmq.managed import IBMQJobManager
 
-class EntanglementTest:
-    @staticmethod
-    def create_bell_state():
-        """Creates two qubit bell state
+def create_bell_state():
+      """Creates two qubit bell state
```

Comment:
indentation tab size should be 4 to be consistent. I think other parts of this file should be reindented.